aws_lambda_powertools/metrics/metric.py

```diff
@@ -109,11 +109,11 @@ def single_metric(name: str, unit: MetricUnit, value: float, namespace: str = No
     SchemaValidationError
         When metric object fails EMF schema validation
     """
-    metric_set = None
+    metric_set: Optional[Dict] = None
```

Comment:
We can get rid of this - It was added in error. `metric_set` being created within `try` block will also be available in `finally`.

Reply:
If we do this then PyCharm complains and wants it to be a global

<img width="823" alt="Screen Shot 2021-03-25 at 1 09 38 AM" src="https://user-images.githubusercontent.com/5442469/112439826-cf2bb380-8d06-11eb-8c2a-5dab39ad556b.png">